packages/components/tooltip/src/trigger.vue

```diff
@@ -50,7 +50,18 @@ const stopWhenControlledOrDisabled = () => {
 const trigger = toRef(props, 'trigger')
 const onMouseenter = composeEventHandlers(
   stopWhenControlledOrDisabled,
-  whenTrigger(trigger, 'hover', onOpen)
+  whenTrigger(trigger, 'hover', (e) => {
+    onOpen(e)
+
+    if (props.focusOnTarget && e.target) {
+      nextTick(() => {
+        focusElement(e.target as HTMLElement, {
+          preventScroll: true,
+          focusVisible: false,
```

Comment:
<img width="1053" height="299" alt="Image" src="https://github.com/user-attachments/assets/7b293aa1-33d7-4a7f-9695-968e363974c6" />

It seems that most browsers do not support the `focusVisible` option. Can we remove this option? Will this have any impact?